Readme.md

```diff
@@ -1,23 +1,82 @@
-# Microservice Setup
-BISKOT API is the new micro service responsible for managing the cart of an e-commerce website.
+# Biskop 
+Biskop API is micro service responsible for managing the cart of an e-commerce website.
 
-## Your Mission should you choose to accept it, is to:
+Before attempting to crunch the code, we strongly recommend you to carefully read each of the sections below:
 
-* Implement the endpoints defined in the swagger [biskot.yaml](contract/biskot.yaml), api model classes could be generated using openapi-generator-maven-plugin configured in pom.xml
-* Setup the development best practices
+## Exercise goal
 
-## Cart business rules:
+* Implement the endpoints defined in the [biskot.yaml](contract/biskot.yaml) api model. To better understand the model, we suggest you open the `yaml` file in a fancy UI available [here](https://editor.swagger.io/).
+
+To give you an idea of what is the expected result from each endpoint looks like, below some examples:
+
+> #### action: POST /carts
+> response http code: 200
+
+> #### action: GET /carts/{cartId}
+> response http code: 200
+>
+> response body:
+>```json
+>{
+>  "id": 1,
+>  "products": [
+>    {
+>      "id": 1,
+>      "label": "Déodorant Spray 200ml Ice Dive ADIDAS",
+>      "quantity": 1,
+>      "unit_price": 2.00,
+>      "line_price": 2.00
+>    }
+>  ],
+>  "totalPrice": 0
+>}
+>>```
+
+> #### action: PUT /carts/{cartId}/products
+> request body:
+> ```json
+> {
+>    "id": 1,
+>    "quantity": 2
+>  }
+>```
+> response http code: 200
+>
+> response body:
+>```json
+>{
+>  "id": 1,
+>  "products": [
+>    {
+>      "id": 1,
+>      "label": "Déodorant Spray 200ml Ice Dive ADIDAS",
+>      "quantity": 3,
+>      "unit_price": 2.00,
+>      "line_price": 6.00
+>    }
+>  ],
+>  "totalPrice": 0
+>}
+>>```
```

Comment:
Add product to cart as described in contract respond with 200 and empty body when addition succeeded
```suggestion
```